kordac/tests/SmokeTests.py

```diff
@@ -65,6 +68,9 @@ def test_compile_files(self):
                 self.assertTrue(len(result.html_string) > 0)
 
     def test_compile_files_custom(self):
```

Comment:
The files tested are straight out of csfieldguide so it does not test everything but that in a generic usecase it will still work.